auth/src/main/java/org/aerogear/auth/credentials/OIDCCredentials.java

```diff
@@ -67,8 +145,21 @@ public boolean isAuthorized() {
      * Returns stringified JSON for the OIDCCredential.
      * @return Stringified JSON OIDCCredential
      */
-    public String serialise() {
-        return this.authState.jsonSerializeString();
+    public String serialize() throws JSONException {
```

Comment:
How is serialize used?

Reply:
Personally I would remove the JSONException from the method declaration and rethrow it as a runtime exception and add that clue to the documentation.

Reply:
@secondsun serialize is used when persisting the auth state in the AuthStateManager.